src/dev/build/tasks/clean_tasks.ts

```diff
@@ -200,3 +197,16 @@ export const CleanEmptyFolders: Task = {
     ]);
   },
 };
+
+export const DeleteBazelPackagesFromBuildRoot: Task = {
+  description:
+    'Deleting bazel packages outputs from build folder root as they are now installed as node_modules',
+
+  async run(config, log, build) {
+    const bazelPackagesOnBuildRoot = (await discoverBazelPackages())
+      .filter((pkg) => !pkg.isDevOnly())
```

Comment:
Why keep dev packages in the build root?

Reply:
To avoid trying to delete something that won't be there as we don't initially copy dev packages. But I think its safe to remove this anyway 👍